app/Core/Repository/ClientRepository.cs

```diff
@@ -109,7 +109,7 @@ public Client GetClientInformation(Client client)
 
             dt = upgradeFile.Load("SELECT * FROM vwclient WHERE clientId=@Id;", parameters);
             if (dt.Rows.Count == 0)
-                throw new Exception("Empty DataTable");
```

Comment:
Please indent the Exception after the if condition, as it should only be executed when the returned datatable is empty.